client/my-sites/marketing/connections/service.jsx

```diff
@@ -218,24 +218,13 @@ export class SharingService extends Component {
 	};
 
 	/**
-	 * Create or update the connection
+	 * Create the connection for the site
 	 *
 	 * @param {number} keyringConnectionId Keyring conneciton ID.
-	 * @param {number} externalUserId      Optional. User ID for the service. Default: 0.
+	 * @param {number} externalUserId      User ID for the service.
 	 */
-	createOrUpdateConnection = ( keyringConnectionId, externalUserId = 0 ) => {
```

Comment:
May be leave the `createOrUpdateConnection` as well till we get rid of the feature flag?

Reply:
Sure!